kibana.dev.yml

```diff
@@ -8,7 +8,7 @@ logging:
   appenders:
     file:
       type: file
-      fileName: ./kibana.log
+      fileName: /var/log/kibana.log
```

Comment:
think I'll revert this change, since I don't use this in the GCP deployment. doesn't work with 7.x branches due to new logging settings , so I'll leave a separate note